openid-4-verifiable-presentations-1_0.md

```diff
@@ -261,6 +261,9 @@ This specification defines the following new parameters:
 
 A public key to be used by the Wallet as an input to the key agreement to encrypt Authorization Response (see (#jarm)). It MAY be passed by the Verifier using the `jwks` or the `jwks_uri` claim within the `client_metadata` or `client_metadata_uri` request parameter.
 
+`create_request_uri`: 
+: OPTIONAL. A string containing an HTTPS URL pointing to a resource where the Wallet MUST request the creation of a request object as defined in [@RFC9101]. The details of this endpoint are defined in (#create_request_uri). This parameter MUST only be combined with `client_id`, `state`, `client_id_scheme`, and any client id scheme specific parameter. It SHOULD be sent in a signed authorization request. 
```

Comment:
I think if the RP is registered in some trust framework (aka client_id_scheme), which the wallet can access using its own configured information, then a signed request is not needed if the wallet can determine that the asserted client_id is trustworthy and the wallet is not relying on any other unsigned information sent by the RP in the authz request i.e. the trust framework can provide the wallet with all the necessary metadata in a trustworthy manner.

i.e. it does not help a fake RP to pretend to be a trustworthy one, if the wallet sends the VP to the trustworthy RP's endpoint (except perhaps as a complex DOS attack on the trustworthy RP?)

Reply:
Yes this was my understanding too. If there is nothing more to it than this then I think we should remove this sentence, since it isn't helpful to say the request "SHOULD" be signed without any further context. If we haven't already we should probably add text to the effect of what you just described David, i.e. how a wallet can trust signed requests and when it can trust unsigned requests.

Reply:
added some context - the primary purpose of the signature is to authenticate the verifier early in the process to prevent abuse of the request URI callback for tracking.